settings.d/remote_execution_ssh.yml.example

```diff
@@ -10,3 +10,7 @@
 # for new data leave empty to use the runner's default
 # (1 second for regular, 60 seconds with async_ssh enabled)
 # :runner_refresh_interval:
+
+# Defines the verbosity of logging coming from Net::SSH
+# one of :debug, :info, :warn, :error, :fatal
```

Comment:
If I remember properly, the `ssh_log_level` should be higher than the `smart_proxy_dynflow_core` log level. Perhaps we should state it here as well